web/src/beta/features/Dashboard/ContentsContainer/Members/index.tsx

```diff
@@ -76,37 +115,60 @@ const Members: FC<{ currentWorkspace?: Workspace }> = ({
 
 export default Members;
 
-const Wrapper = styled("div")(() => ({
+// styled components
+const Wrapper = styled("div")({
   display: "grid",
-  gridTemplateRows: "auto 1fr",
-  boxSizing: "border-box",
+  gridTemplateRows: "auto auto 1fr",
   height: "100%"
-}));
+});
+
+const HeaderWrapper = styled("div")({
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "center"
+});
```

Comment:
Doulble check the style with design, should have padding, and adjust the padding of it's children also.